### ISSUE TITLE
Exception when trying to share Youtube Video to TV

### ISSUE THREAD

**Issue by ein-bandit** [tobexyz/yaacc-code#76]
device i run yaacc on: pixel 4a, Graphene OS (Android 13)
device i want to cast to: LG TV

Problem: App crashes when sharing Youtube link to yaacc.

Log:
```
type: crash
osVersion: google/sunfish/sunfish:13/TQ2A.230305.008.C1/2023040400:user/release-keys
package: de.yaacc:40000
process: de.yaacc
processUptime: 268 + 514 ms

java.lang.RuntimeException: Unable to start activity ComponentInfo{de.yaacc/de.yaacc.browser.TabBrowserActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3642)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3779)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:138)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2304)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7891)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
	at de.yaacc.browser.TabBrowserActivity.onCreate(TabBrowserActivity.java:187)
	at android.app.Activity.performCreate(Activity.java:8388)
	at android.app.Activity.performCreate(Activity.java:8367)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1422)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3623)
	... 14 more

```

**Comment by tobexyz** [comment]
Thank you for your report. I'll have a look at it and try to reproduce

**Comment by tobexyz** [comment]
hopefully fixed in upcoming release 4.0.1 :-)
